samples/csharp_dotnetcore/49.qnamaker-all-features/appsettings.json

```diff
@@ -1,7 +1,10 @@
 {
   "MicrosoftAppId": "",
   "MicrosoftAppPassword": "",
-  "QnAKnowledgebaseId": "",
-  "QnAEndpointKey": "",
-  "QnAEndpointHostName": ""
+  "QnAKnowledgebaseId": "135099bb-f440-4832-be2b-9570bd530215",
+  "QnAEndpointKey": "670bd815e29b4e5ba4e4dccdfb8dd21e",
+  "QnAEndpointHostName": "https://qna-v2-mrc-demo.cognitiveservices.azure.com/qnamaker/v5.0-preview.1",
+  "QnAServiceType": "v2",
+  "EnablePreciseAnswer": "false",
```

Comment:
make to check-in this file with 3 additional fields added + empty values for all